db/migrate/20171017143139_add_upload_documents_to_medicaid_applications.rb

```diff
@@ -0,0 +1,23 @@
+class AddUploadDocumentsToMedicaidApplications < ActiveRecord::Migration[5.1]
+  def up
+    add_column :medicaid_applications, :upload_documents, :boolean
+
+    # More info on adding column with default value
+    # github.com/ankane/strong_migrations#adding-a-column-with-a-default-value
+    #
+    add_column \
+      :medicaid_applications,
+      :documents,
+      :string,
+      array: true
+
+    change_column_default :medicaid_applications, :documents, []
+
+    commit_db_transaction
```

Comment:
ooh! I feel like I've been missing this line when adding columns with default values (I guess I read the docs in strong migrations but not closely enough)

Reply:
yeah, this is interesting!